server/rest/src/main/java/org/infinispan/rest/RestRequestHandler.java

```diff
@@ -85,6 +85,7 @@ public void channelRead0(ChannelHandlerContext ctx, FullHttpRequest request) thr
          // Ensure that the authorization header, if needed, has not changed
          String authz = request.headers().get(HttpHeaderNames.AUTHORIZATION);
          if (Objects.equals(authz, authorization)) {
+            restRequest.setSubject(subject);
```

Comment:
@gustavonalle @tristantarrant this little thing has been driving me crazy :))) and I think this explains why I could not create a cache from the console but you told me you could. Tests with auth enabled had been like crazy for this

Reply:
Not only that, but I think there is something wrong further down:

`
            subject = restRequest.getSubject();
            authorization = restRequest.getAuthorizationHeader();
            restRequest.setSubject(subject);
`
I think the `setSubject` can be removed

Reply:
where further down